### ISSUE TITLE
Error when using exq_ui with exrm

### ISSUE THREAD

**Issue by jasondew** [akira/exq_ui#29]
First of all, thanks for your work on `exq` and `exq_ui`, it's really great so far! I do have one hangup, though. I'm using exrm for deployment, since that was recommended, and I have `exq` and `exq_ui` in my `applications` function, also as recommended. However, when I try to start the server, I get:

```
Application exq exited: Exq.start(:normal, []) returned an error: already started: #PID<0.1350.0>
```

I've tried all permutations of including/disincluding `exq` and `exq_ui` as well as different configs to no effect. Can you provide any help?

**Comment by jasondew** [comment]
I switched over to use https://github.com/bitwalker/distillery and everything works now. Closing!

**Comment by isaksky** [comment]
@jasondew What settings did you use to make it work with distillery? Having the same issue.

**Comment by jasondew** [comment]
Sorry @isaksky, I don't remember anything in particular that I did to make this work other than switching and including both `exq` and `exq_ui` in my `applications` list. I also have 

```elixir
config :exq_ui,
  server: false
```

in my `config/prod.exs`. Hopefully this helps somehow!

**Comment by davidefedrigo** [comment]
Same issue here.

I "fixed" switching back to distillery `~> 0.9.0` from `~> 0.9`.

I kept:
```

**Comment by isaksky** [comment]
@jasondew makes sense, thanks. Did not work for me, unfortunately.

@davidefedrigo interesting. I tried to do the same, but looks like I need some of the newer settings.

I'll try to mess with the exq_ui code.

**Comment by isaksky** [comment]
I was able to get it to start with this commit:

https://github.com/isaksky/exq_ui/commit/a80595d3800855ace9a390d5c8c62b626111497f

**Comment by akira** [comment]
@isaksky Thanks for reporting, I will take a look.  I think I know what's causing the issue.

**Comment by akira** [comment]
@isaksky it should be fixed now, let me know if it doesn't work

**Comment by akira** [comment]
Published 0.8.3 with this fix

**Comment by isaksky** [comment]
@akira I tried this commit out, and it fixed this issue, thanks!

I had some other issues regarding asset paths, seemingly with the new changes, but it may be unrelated, or on my end. I'll let you know if I find out more.
